pydantic_ai_slim/pydantic_ai/_parts_manager.py

```diff
@@ -72,6 +72,7 @@ def handle_text_delta(
         *,
         vendor_part_id: VendorId | None,
         content: str,
+        model_profile: ModelProfile,
         extract_think_tags: bool = False,
```

Comment:
I suggested below changing the field on `ModelProfile` to a single `tuple[str, str]`. Instead of an extra field to pass them here, I think we can reuse `extract_think_tags` if we make it `tuple[str, str] | None = None`